README.md

```diff
@@ -289,18 +289,40 @@ Just as with React, but assuming Angular instead :)
 
 As a bootstrap for the application we need something like:
 
+```typescript 
+@Injectable()
```

Comment:
I think the annotation is not needed since you are using an explicit provider for this dependency. Consider adding a comment clarifying that this is where DI would be wired.